docs/consume-packages/includes/restore-nuget-exe-cli.md

```diff
@@ -1,13 +1,21 @@
-Use the [restore](../../reference/cli-reference/cli-ref-restore.md) command, which downloads and installs any packages missing from the *packages* folder.
+The NuGet CLI [restore](../../reference/cli-reference/cli-ref-restore.md) command downloads and installs any packages that are missing from the packages directory.
 
-For projects migrated to PackageReference, use [msbuild -t:restore](../package-restore.md#restore-using-msbuild) to restore packages instead.
+Like `install`, the `restore` command only adds packages to disk, but doesn't change the project's dependencies. To change project dependencies, modify the *packages.config* file manually, then use `restore`.
```

Comment:
> To change project dependencies, modify the *packages.config* file manually, then use `restore`.

Customers will not be successful if they take this advice. Editing the packages.config file and then restoring will cause NuGet to download and extract the package to disk. However, the project won't actually use the package in any way. The list of changes to the project file is complex and both project and package dependant, so it's much too complex to document for customers to try to do manually. It's necessary to install the package in Visual Studio so that the project updates are automated.

Hand editing the packages.config file is appropriate for packages.config files that are not associated with projects, however.

Reply:
@zivkan - I've removed the advice to hand-edit the packages.config file from both the Install and Restore sections.